src/video/color.hpp

```diff
@@ -81,6 +82,12 @@ class Color final
                  static_cast<float>(a) / 255.0f);
   }
 
+  static std::optional<Color> deserialize_color_from_rgb(const std::string& rgb_string);
+  static std::optional<Color> deserialize_color_from_hex(const std::string& hex_string);
+
+  static std::string serialize_color_to_rgb(const Color& color);
+  static std::string serialize_color_to_hex(const Color& color);
```

Comment:
Just suggestions, change if you feel like it:
1. `_color_` doesn't seem needed in the function names.
2. The serialize functions could be non-`static`, but that doesn't matter much.